src/pages/index.js

```diff
@@ -21,73 +21,229 @@ import {
   profilePicture,
   openModal,
   closePopup,
+  profileAvatarEditButton,
   options,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopUpWithConfirmation.js";
 
 // Data
 const initialCards = [
   {
     name: "Yosemite Valley",
     description: "Yosemite Valley",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
+    _id: "card1",
   },
   {
     name: "Lake Louise",
     description: "Lake Louise",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
+    _id: "card12",
   },
   {
     name: "Bald Mountains",
     description: "Bald Mountains",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
+    _id: "card123",
   },
   {
     name: "Latemar",
     description: "Latemar",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
+    _id: "card4",
   },
   {
     name: "Vanoise National Park",
     description: "Vanoise National Park",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
+    _id: "card5",
   },
   {
     name: "Lago di Braies",
     description: "Lago di Braies",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
+    _id: "card6",
   },
 ];
 //
+// API initialization
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f10ef152-cb9d-40bf-b92c-7c27e5b2bf36",
+    "Content-Type": "application/json",
+  },
+});
+
+// User Info
+const userInfo = new UserInfo({
+  profileTitle: document.querySelector(".profile__title"),
+  profileDescription: document.querySelector(".profile__description"),
+  profilePicture: document.querySelector(".profile__image"),
+});
+
+// Form Validators
```

Comment:
Please, delete `this.toggleButtonState();` from line 69 in  `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.

Reply:
* You haven't deleted something https://github.com/Alexisct2212/se_project_aroundtheus/pull/1#discussion_r1744534725